src/SortAlgorithms.java

```diff
@@ -0,0 +1,130 @@
+public class SortAlgorithms {
+
+    /*
+    1. bubble sort
+    2. selection sort
+    3. insertion sort
+    4. merge sort
+    5. quick sort
+
+//    6. index sort
+     */
+    //TODO:grgthyh
+    void func1(int n) {
```

Comment:
Delete lines from current line to line 35, they're not related to sort algorithms